### PR TITLE
Add train iterations to dreambooth api

### PR DESCRIPTION
**Pull request by stripuramallu3**

### CODE REVIEW

File: textual-inversion/get-project.mdx [hunk 1]
```diff
@@ -31,6 +31,7 @@ curl -X 'GET' \
     "node": {
         "id": "047b6913-fd3d-4b0d-8dca-349f786a40a9",
         "init_word": "bridge",
+        "train_iterations: 3000,
```
Comment (AmanKishore) [inline textual-inversion/get-project.mdx:34]: "train_iterations: 800,

Comment (AmanKishore) [inline textual-inversion/get-project.mdx]:
```suggestion
        "train_iterations: 800,
```

File: textual-inversion/get-projects.mdx [hunk 2]
```diff
@@ -26,6 +26,7 @@ curl -X 'GET' \
         "node": {
             "id": "047b6913-fd3d-4b0d-8dca-349f786a40a9",
             "init_word": "bridge",
+            "train_iterations: 3000,
```
Comment (AmanKishore) [inline textual-inversion/get-projects.mdx]:
```suggestion
            "train_iterations: 800,
```

File: textual-inversion/get-project.mdx [hunk 3]
```diff
@@ -31,6 +31,7 @@ curl -X 'GET' \
     "node": {
         "id": "047b6913-fd3d-4b0d-8dca-349f786a40a9",
         "init_word": "bridge",
+        "train_iterations: 800,
```
Comment (stripuramallu3) [inline textual-inversion/get-project.mdx]:
```suggestion
        "train_iterations": 800,
```

File: textual-inversion/get-projects.mdx [hunk 4]
```diff
@@ -26,6 +26,7 @@ curl -X 'GET' \
         "node": {
             "id": "047b6913-fd3d-4b0d-8dca-349f786a40a9",
             "init_word": "bridge",
+            "train_iterations: 800,
```
Comment (stripuramallu3) [inline textual-inversion/get-projects.mdx]:
```suggestion
            "train_iterations": 800,
```

File: textual-inversion/create-project.mdx [hunk 5]
```diff
@@ -16,6 +16,10 @@ Creates a textual inversion project
 Describes the word
 </ParamField>
 
+<ParamField body="train-iterations" type="string">
```
Comment (stripuramallu3) [inline textual-inversion/create-project.mdx]:
```suggestion
<ParamField body="train-iterations" type="string" default="800">
```

**Review by AmanKishore — APPROVED** [review]
Lgtm

**Pull request merged**